tests/performance-tests/atala-performance-tests-k6/README.md

```diff
@@ -10,27 +10,30 @@
 
 Clone the generated repository on your local machine, move to the project root folder and install the dependencies defined in [`package.json`](./package.json)
 
+*NOTE*: The Project has a dependency on `input-output-hk/prism-typescript-client` which is a private repository.
```

Comment:
:::note

The project is dependent on `input-output-hk/prism-typescript-client` which is a private repository.